src/payment/payment-action-creator.ts

```diff
@@ -31,7 +31,15 @@ export default class PaymentActionCreator {
                     ))
                 )
         ).pipe(
-            catchError(error => throwErrorAction(PaymentActionType.SubmitPaymentFailed, error))
+            catchError(error => {
```

Comment:
I'd probably just leave `PaymentActionCreator` as it is and let the new `getGoogleRecaptchaToken` function you've created to parse the error response instead.